install-linux.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#create tile38 user
+useradd -m -d /var/lib/tile38 -s /sbin/nologin -U tile38
+
+#copy binary file to bin path
+cp -r tile38-server tile38-cli tile38-benchmark    /usr/bin/
+
+#create PID dir
+mkdir -p /run/tile38
+chown -R tile38:tile38  /run/tile38
+chmod -R 755 /run/tile38
+
+#create PID file
+touch /run/tile38/tile38.pid
+chown -R tile38:tile38  /run/tile38/tile38.pid
+chmod -R 664 /run/tile38/tile38.pid
+
+#create systemd file
+cat > /usr/lib/systemd/system/tile38.service << 'EOF'
```

Comment:
👎 Many systems still use init.d

Also `/user/lib/systemd` is reserved for distro-installed packages. Use `/etc/systemd/system` instead

Reply:
yes. I have changed it.